library/pod-security-policy/proc-mount/src_test.rego

```diff
@@ -142,6 +157,10 @@ input_parameters_default = {
      "procMount": "Default"
 }
 
+input_parameters_default_lower = {
+     "procMount": "default"
```

Comment:
can you also add a test for `parameters: null` and no parameter in input?

Reply:
Added code to handle null or missing param, and tests for those cases.